backend/app/api/endpoints/project_automations.py

```
@@ -0,0 +1,131 @@
+    ProjectAutomationUpdate,
+    ProjectAutomationView,
+)
+from app.services.project_automations import project_automation_service
+
+router = APIRouter()
+
+
+@router.get("/{project_id}/automations", response_model=list[ProjectAutomationView])
+def list_automations(
+    project_id: str,
+    db: Session = Depends(get_db),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)